src/ansible_compat/runtime.py

```diff
@@ -199,6 +203,35 @@ def install_collection(
             _logger.error(msg)
             raise InvalidPrerequisiteError(msg)
```

Comment:
As for the failure with Ansible 2.9 in tests, maybe make the argument an explicit path? as in pass `./acme-goodies-1.0.0.tar.gz`?

Reply:
Sorted with full path.